python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -376,11 +377,14 @@ def add_output_metadata(
     ) -> None:
         """Add metadata to one of the outputs of an op.
 
-        This can only be used once per output in the body of an op. Using this method with the same output_name more than once within an op will result in an error.
+        This can be invoked multiple times per output in the body of an op. If the same key
```

Comment:
"If the same key is passed multiple times, the value associated with last call will be used" is a little more specifiy